### PR TITLE
Fix pre-commit isort hook failure for benchmarks subdir.

### PR DESCRIPTION
**Pull request by arnaudgelas**
Rework #5398

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

File: benchmarks/test_basic_parity.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
 
 import numpy as np
 import pytest
-import torch
 from tqdm import tqdm
 
+import torch
```
Comment (Borda) [inline benchmarks/test_basic_parity.py:21]: is this correct as `torch` is external lib?

**Review by akihironitta — COMMENTED** [review]
@arnaudgelas It seems that all the changes in this PR don't comply with our current isort config. You might be accidentally using a different config...?

**Comment by akihironitta** [comment]
@arnaudgelas That's strange... Which version of `isort` are you using? I'm using `5.7.0`, and it works fine.

**Comment by arnaudgelas** [comment]
@akihironitta Thanks! You gave me an excellent pointer to fix it. The main problem is pre-commit was relying on `isort` found on your system. In my case it was an extremely old one (and not the one installed via provided requirements).

 see #5408

**Comment by akihironitta** [comment]
@arnaudgelas You're welcome!

**Pull request closed without merging**